pkg/api/volume_resp.go

```diff
@@ -76,15 +76,15 @@ type VolumeStatusResp struct {
 type VolumeCreateResp VolumeInfo
 
 // VolumeGetResp is the response sent for a volume get request.
-/* VolumeGetResp can also be filtered based on query parameters
+/*
+VolumeGetResp can also be filtered based on query parameters
 sent along with volume list/info api.
 Query Parameters can be either metadata key/value/both key and value.
-Example of API request-
-		1- /volumes -
-					a- GET 'http://localhost:24007/v1/volumes?key={keyname}&value={value}
-					b- GET 'http://localhost:24007/v1/volumes?key={keyname}'
-					c- GET 'http://localhost:24007/v1/volumes?value={value}'
-Note - Cannot use query parameters or cli flags if volname is also supplied.
+Example of API request
+        - GET 'http://localhost:24007/v1/volumes?key={keyname}&value={value}
```

Comment:
Missing closing quote. You can actually remove quotes altogether. Here and in examples below.